### PR TITLE
Change default multiplication behavior

### PR DESCRIPTION
**Pull request by ZibingZhang**
# Overview

By default, multiplication should have a `\cdot`.

# Details

Based on [this discussion](https://github.com/google/latexify_py/issues/89#issuecomment-1344967636), it's clear that not having `\cdot` is the exception rather than the rule.

As a precursor to work implementing the exception behavior, we should first move the default behavior to include a `\cdot`, in order to make the future PR easier to follow. We basically split the that issue into two

1. first change default behavior
2. TODO: implement exceptions

# References

https://github.com/google/latexify_py/issues/89

# Blocked by

None

### CODE REVIEW

File: src/latexify/codegen/expression_codegen.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ class BinOpRule:
         operand_left=BinOperandRule(force=True),
         operand_right=BinOperandRule(wrap=False),
     ),
-    ast.Mult: BinOpRule("", " ", ""),
+    ast.Mult: BinOpRule("", r" \cdot ", ""),
```
Comment (odashi) [inline src/latexify/codegen/expression_codegen.py:125]: Revise `MatMult` as well.

Comment (ZibingZhang) [reply src/latexify/codegen/expression_codegen.py:125]: revised

**Comment by odashi** [comment]
As per comments in #162, this pull request is merged after that change.

**Review by odashi — APPROVED** [review]

**Comment by ZibingZhang** [comment]
Think this is good to merge as well

**Review by odashi — APPROVED** [review]

**Pull request merged**